client/components/download-link/renderers/docx.js

```diff
@@ -322,6 +359,10 @@ const renderField = (doc, field, values) => {
     return renderSpeciesSelector(doc, values, value);
   }
 
+  if (field.type === 'animal-quantities') {
+    return renderAnimalQuantities(doc, values);
+  }
```

Comment:
Does this need to move below the `renderNull` block below?

Reply:
No.

Reply:
/shrug.gif